.travis.yml

```diff
@@ -257,6 +257,30 @@ jobs:
         - make image-build-scorecard-test-kuttl
         - make image-push-scorecard-test-kuttl
 
+    # Build and deploy arm64 scorecard-test-kuttl docker image
+    - <<: *deploy
+      name: Docker image for scorecard-test-kuttl (arm64)
+      arch: arm64
+      script:
+        - make image-build-scorecard-test-kuttl
+        - make image-push-scorecard-test-kuttl
+
+
+    # Build and deploy ppc64le scorecard-test-kuttl docker image
+    - <<: *deploy
+      name: Docker image for scorecard-test-kuttl (ppc64le)
+      arch: ppc64le
+      script:
+        - make image-build-scorecard-test-kuttl
+        - make image-push-scorecard-test-kuttl
+
+    # Build the amd64 custom-scorecard-tests docker image
+    - <<: *deploy
+      name: Docker image for custom-scorecard-tests (amd64)
+      arch: amd64
+      script:
+        - make image-build-custom-scorecard-tests
```

Comment:
This will only run after a PR has merged. Can you add this to the integration/subcommand test job so it gets built during PR tests?

Reply:
doh!  updated it to be in the the other test job.

Reply:
https://travis-ci.org/github/jmccormick2001/operator-sdk/builds/719760518